tslib/src/functions/check-validator/validators/answer-count.validator.ts

```diff
@@ -0,0 +1,16 @@
+import { ISubmittedCheckValidator, CheckValidationResult } from './validator-types'
+
+export class AnswerCountValidator implements ISubmittedCheckValidator {
+  validate (check: any): CheckValidationResult {
+    if (check.answers === undefined) {
+      return {
+        message: 'no answers property found'
+      }
+    }
+    if (check.answers.length < 25) {
```

Comment:
Hard-coding the expected answer length as a magic number?   I would rather see the expected answer count determined dynamically, in a named variable.

Reply:
we briefly covered this in the session we had about marking / forms.  i will move this to config, same as admin app